### PR TITLE
Updated dates for Python added definitions and codeblocks

### LINKED ISSUE #5392: [Edit] Python Date
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

We would like to edit the `Date` concept entry under Python. The entry lives in `content/python/concepts/dates/terms/date/date.md`.

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add an Example section in this entry with a code example and its output.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by tanmaytare**
### Issue Solved
Closes #5392 

Made changes added example for dates.md

### Type of Change

- Adding a new example
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [✓] All writings are my own.
- [✓] My entry follows the Codecademy Docs style guide.
- [✓] My changes generate no new warnings.
- [✓] I have performed a self-review of my own writing and code.
- [✓] I have checked my entry and corrected any misspellings.
- [✓] I have made corresponding changes to the documentation if needed.
- [✓] I have confirmed my changes are not being pushed from my forked `main` branch.
- [✓] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [✓] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by SaviDahegaonkar** [comment]
Hey @tanmaytare , This PR does not mention the issue to which this PR relates to, due to which I find this PR invalid, you can raise new PR mentioning the same.

Thanks.

**Comment by mamtawardhani** [comment]
@SaviDahegaonkar Please review this at the earliest?

File: content/python/concepts/dates/terms/date/date.md [hunk 1]
```diff
@@ -39,3 +39,29 @@ meeting_date = datetime.date(2021, 10, 5)
 
 print(meeting_date)
 ```
+
+## Example
+
+Here's a more detailed example of using datetime.date() to create a date object and access its components:
+
+```python
```
Comment (SaviDahegaonkar) [inline content/python/concepts/dates/terms/date/date.md:47]:
```suggestion
```py
```

File: content/python/concepts/dates/terms/date/date.md [hunk 2]
```diff
@@ -39,3 +39,29 @@ meeting_date = datetime.date(2021, 10, 5)
...
+month = example_date.month
+day = example_date.day
+
+print("Complete Date:", example_date)
+print("Year:", year)
+print("Month:", month)
+print("Day:", day)
+```
+
+```Output:
```
Comment (SaviDahegaonkar) [inline content/python/concepts/dates/terms/date/date.md:62]:
```suggestion
The above example demonstrates how to create a date object in Python using the `date` class:
```shell 
```

**Review by SaviDahegaonkar — APPROVED** [review]

**Review by Sriparno08 — APPROVED** [review]
Looks good, @tanmaytare!

**Pull request merged**